internal/sys/types.go

```diff
@@ -646,6 +646,26 @@ func LinkCreateIter(attr *LinkCreateIterAttr) (*FD, error) {
 	return NewFD(int(fd))
 }
 
+type LinkCreateKprobeMultiAttr struct {
+	ProgFd           uint32
+	TargetFd         uint32
+	AttachType       AttachType
+	Flags            uint32
```

Comment:
@joamaki somewhat OT, but why does kprobe_multi not use the existing flags field?

Reply:
@lmb I think you mean @olsajiri :wink: